Vostok.ClusterClient.Transport.Tests/Functional/Common/SendFailureTests.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Net.Sockets;
+using FluentAssertions;
+using NUnit.Framework;
+using Vostok.Clusterclient.Core.Model;
+using Vostok.Clusterclient.Transport.Tests.Helpers;
+using Vostok.Commons.Environment;
+using Vostok.Commons.Threading;
+
+namespace Vostok.Clusterclient.Transport.Tests.Functional.Common
+{
+    internal abstract class SendFailureTests<TConfig> : TransportFunctionalTests<TConfig>
+        where TConfig : ITransportTestConfig, new()
+    {
+        [Test]
+        public void Should_return_SendFailure_code_when_socket_closed_while_sending_large_body()
+        {
+            if (!RuntimeDetector.IsDotNetCore21AndNewer)
```

Comment:
just failing with some CURL error: https://github.com/bymse/clusterclient.transport/runs/3274602331?check_suite_focus=true#step:10:503  
but net core 2.0 will be out of support in a few weeks